scalding-core/src/main/scala/com/twitter/scalding/FileSource.scala

```diff
@@ -419,7 +419,8 @@ abstract class FixedPathSource(path: String*) extends FileSource {
   def localPaths = path.toList
   def hdfsPaths = path.toList
 
-  override def toString = getClass.getName + path
+  // The collection type of `path` causes equality issues in JobTest
```

Comment:
@ianoc That's my understanding as well, and yes, I've confirmed that toString is the primary means of [doing equality check](https://github.com/twitter/scalding/blob/develop/scalding-core/src/main/scala/com/twitter/scalding/FileSource.scala#L424) (hence the [comment here](https://github.com/twitter/scalding/blob/develop/scalding-core/src/main/scala/com/twitter/scalding/JobTest.scala#L63)). 

So, you'd like me to append that to the comment?